### PR TITLE
moving package managers to rules-docker repo

### PR DESCRIPTION
**Pull request by nlopezgi**
https://github.com/bazelbuild/rules_docker/pull/970 moved most rules under package_managers to rules_docker repo.
This is because dependency management for repos that depend on package_managers was becoming complicated. Moreover, making sure changes to rules_docker did not cause regressions of package_managers was a not trivial task. Lastly, we expect these rules will get much more usage if they are in rules_docker repo.

This PR updates base-images-docker to use package_managers in rules_docker.
Change includes creating facades so that users that still depend on old location of rules in this repo can migrate safely.

Not refactored (from package_mangers in this repo to rules_docker):
- bootstrap_image: will remain in this repo as functionality is only expected to be used here
- security_check: will be migrated to rules_docker in an upcoming PR.

This PR also changes the sub-projects under ubuntu1604, ubuntu1804, debian9 and centos7 directories to use util/package_manager rules in rules_docker repo as well, and remove dependencies on base-images-docker if needed.

This PR also fixes issues related to compatibility with latest Bazel version (e.g., fix uses of allow_single_file, add .bazelrc files to --add host_force_python=PY2 flags.

### CODE REVIEW

File: centos7/.bazelrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```
Comment (xingao267) [inline centos7/.bazelrc]: License here should be the google wide one.

File: ubuntu1604/WORKSPACE [hunk 2]
```diff
@@ -22,42 +22,25 @@ load(
...
-    sha256 = "1a035f4e9c21e48668e09b327f89bd8197feb42b82d2b6904913c655f27a845a",
-    strip_prefix = "rules_docker-bb6d6606a6be348115af3552662799fd6d851a88",
-    urls = ["https://github.com/bazelbuild/rules_docker/archive/bb6d6606a6be348115af3552662799fd6d851a88.tar.gz"],
+    sha256 = "b16afde01b99f8b9a067b98903a5fa1879307f9344ccdbbc4aa796ae657e910f",
+    strip_prefix = "rules_docker-4ea9e6708ec5ac29dc18f991921e2c064969159a",
+    urls = ["https://github.com/bazelbuild/rules_docker/archive/4ea9e6708ec5ac29dc18f991921e2c064969159a.tar.gz"],
 )
 
 http_archive(
     name = "base_images_docker",
```
Comment (xingao267) [inline ubuntu1604/WORKSPACE]: Do you still need this here?

Comment (xingao267) [reply ubuntu1604/WORKSPACE]: nvm. the security check i guess.

File: centos7/.bazelrc (hunk 1 above)
Comment (nlopezgi) [reply centos7/.bazelrc]: fixed

**Pull request merged**